miio/airconditioningcompanion.py

```diff
@@ -212,7 +214,7 @@ def target_temperature(self) -> Optional[int]:
     def swing_mode(self) -> Optional[SwingMode]:
         """Current swing mode."""
         try:
-            mode = int(self.state[5:6])
+            mode = self.state[5:6]
```

Comment:
Should this actually be returning a boolean instead of an enum (and exposing some extra information otherwise, if/when those unknowns get handled)? Otherwise feel free to merge as you will.

Reply:
It's more or less a workaround. I'm collecting some knowledge to provide a proper solution some day.